tests/store_test.go

```diff
@@ -40,7 +40,7 @@ type StoreSuite struct {
 func (s *StoreSuite) TestStoreNew() {
 	doc := NewStoreWithConstructFixture("foo")
 	s.False(doc.IsPersisted())
-	s.False(doc.ID.IsEmpty())
+	s.True(doc.ID.IsEmpty())
```

Comment:
If every Model implements `IDSetter`... then this should be maintained, shouldn't it?